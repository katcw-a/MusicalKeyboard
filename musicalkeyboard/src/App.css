.App {
  display: inline-block;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-body {
  display: inline-display;
  margin: 0;
  font-size: calc(10px + 2vmin);
}

.keyboardRow {
  text-align: center;
}

.Key-inline {
  display: inline-block;
  border: 1px solid black;
  text-align: center;
  height: 85px;
  width: 85px;
  margin-bottom: 5px;
  background-color: #535f63;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
