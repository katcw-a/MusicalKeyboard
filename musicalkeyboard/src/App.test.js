import App from './App';
import React from 'react';
import { shallow } from 'enzyme';

const app = 'Test App'
let wrapped = shallow(<App>{app}</App>);

describe('App', () => {
  it('should render the App componente', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('renders Keyboard in App', () => {
    expect(wrapped.find('Keyboard'));
  });
});