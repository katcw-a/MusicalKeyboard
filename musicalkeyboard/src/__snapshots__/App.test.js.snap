// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App componente 1`] = `
<div
  className="App"
>
  <Keyboard />
</div>
`;
